<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QListWidget" name="listWidget_4"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>user name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="listWidget_2"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>email:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="listWidget">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>age: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="listWidget_3"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>gender:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QListWidget" name="listWidget_5"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>return</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
